### PR TITLE
[DO NOT MERGE - ONLY REVIEW] #1070: Upgrade to Helm v3

### PR DESCRIPTION
**Pull request by oderayi**
- Updated CircleCI config to use Helm v3
- Updated chart versions to 9.3.0
- Consolidated `requirements.yml` and `Chart.yml`
- Updated apiVersion for Charts.yml to v2

### CODE REVIEW

**Comment by bushjames** [comment]
@elnyry what is the timeline for helm version upgrade? This is pretty significant to current users and I suspect they will want time to test it before it becomes a necessity.

**Comment by elnyry** [comment]
@bushjames Currently we're working on making the changes for Helm v3 then, we'll try things out on the dev environments first, to understand compatibility issues with current Helm version (v2) (Example, deploy v3 charts using v2 on cluster setup using v2, though there are no server changes needed for helm v3, etc). 

The plan (Miguel, Steven were I discussed it this morning) is to have two separate repos moving forward, a new repo (incubation) for Helm v3 to identify its behavior and then publish it only after internal validations. However, the plan is to keep the current helm repo intact to support implementations and adopters still using Helm v2 though we may not actively make changes to it, just to support Users already using it (unless its critical security, etc. issues).

Once we have a consensus or a general agreement, we can then move over to Helm v3 and then deprecate the Helm v2 repo at that time.

**Comment by elnyry** [comment]
@oderayi was able to deploy this in AWS dev environments a couple of times with default values and were able to deploy it on our dev2 environment (mojadev2) with the values for dev2.

Below are the results of the Golden path and the results are promising (no issues so far).

However, we'll probably not be merging this and instead move this to a new repo to maintain versions for both helm 2 (for partners,  community members who need it) and helm 3 

image

File: README.md [hunk 1]
```diff
@@ -129,9 +129,9 @@ Expected output:
 
 ## Removing Deployments
 
-`helm del --purge <release-name>`
+`helm uninstall <release-name>`
```
Comment (mdebarros) [inline README.md:132]: Please use `helm del <release-name>` <-- this will match the deployment guide changes for helm v3.

**Review by mdebarros — CHANGES_REQUESTED** [review]
If this PR is still applicable, please pull the latest changes from the master branch and resolve merge conflicts.

Also, see my one suggested comment.

**Comment by mdebarros** [comment]
This issue has been resolved, refer to https://github.com/mojaloop/helm/issues/219 for more information.

**Pull request closed without merging**